frontend/integration-tests/protractor.conf.ts

```diff
@@ -99,6 +99,7 @@ export const config: Config = {
     catalog: ['tests/base.scenario.ts', 'tests/catalog.scenario.ts'],
     operatorHub: ['tests/base.scenario.ts', 'tests/operator-hub/operator-hub.scenario.ts'],
     overview: ['tests/base.scenario.ts', 'tests/overview/overview.scenario.ts'],
+    e2e: ['tests/base.scenario.ts', 'tests/crud.scenario.ts', 'tests/secrets.scenario.ts', 'tests/filter.scenario.ts', 'tests/modal-annotations.scenario.ts', 'tests/environment.scenario.ts', 'tests/overview/overview.scenario.ts', 'tests/source-to-image.scenario.ts', 'tests/deploy-image.scenario.ts', 'tests/performance.scenario.ts', 'tests/olm/descriptors.scenario.ts', 'tests/olm/catalog.scenario.ts', 'tests/olm/etcd.scenario.ts'],
```

Comment:
I'd break this across lines like we do for `all` just below. It'll make it easier to add and remove scenarios.

@jeff-phillips-18 FYI enabling the marketplace tests should be as easy as adding them here when we're ready.